src/keboola/facebook/api/request.clj

```diff
@@ -46,20 +46,21 @@
 
 (defn skip-error-exception? [e]
   (if-let [status (:status e)]
-      (and
-       (<= 400 status 500)
-       (or
-        (re-find #"Media Posted Before Business Account Conversion" (:body e))))))
+    (and
+     (<= 400 status 500)
```

Comment:
I know it's not changed other than spacing, but I needed to comment on this syntax `(<= 400 status 500)`.  mind-blowing :-D